configs/templates/skylab-3.0.0/spack.yaml

```diff
@@ -0,0 +1,95 @@
+spack:
+  concretizer:
+    unify: when_possible
+
+  view: false
+
+  packages:
+    fiat:
+      version: [1.0.0]
+    ectrans:
+      version: [1.0.0]
```

Comment:
Should this be 1.1.0?

Reply:
Argh, yes! One more reason to reinstall everything cleanly (in addition to the crtm update). Thanks for catching this!

Reply:
Actually, it doesn't matter (but makes sense to fix), because the hard requirement is further down in line 39. So all the environments are currently using the correct version. We should be able to remove the duplicate statement later, for now I'll just match the 1.1.0 version from below.

Reply:
skylab-dev looks the same (with version 1.0.0 in ectrans section and 1.1.0 later on in the file)

Reply:
I just fixed all of them (three in total, skylab-3.0.0, skylab-dev, skylab-no-python-dev). Fortunately this information is not used (and we should try to clean up post release), but thanks for spotting this!